src/main/java/com/github/cypher/root/roomcollectionlist/RoomCollectionListPresenter.java

```diff
@@ -15,11 +21,35 @@
 	private Settings settings;
 
 	@FXML
+	private ListView roomCollectionListListView;
+
+	private static final double SERVERLISTCELLHEIGHT=60;
+	private static final double SERVERLISTCELLPADDING_BOTTOM=5;
```

Comment:
Shouldn't it be SERVER_LISTCELL_HEIGHT or at least SERVERLISTCELL_HEIGH. Same with SERVERLISTCELLPADDING_BOTTOM.